package.json

```diff
@@ -25,7 +25,9 @@
     "i18next-browser-languagedetector": "^7.0.0",
     "javascript-time-ago": "^2.5.9",
     "lodash.uniqueid": "^4.0.1",
+    "moment": "^2.29.4",
```

Comment:
please, make localization through date-fns (we already have it). Moment is outdated and doesn't supported
![image](https://user-images.githubusercontent.com/35298264/215425337-1b98c78d-ab4a-429c-ac1d-67feaebbca18.png)

Reply:
done